packages/design-system/src/components/ChartWrapper/index.ts

```diff
@@ -14,33 +14,5 @@
 // You should have received a copy of the GNU General Public License
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 // =============================================================================
-import styled from "styled-components";
-import ProgressBar from "react-customizable-progressbar";
-import { styles } from "../../styles/globalStyleConstants";
-
-export const StyledProgressBar = styled(ProgressBar)`
-  &.RCP {
-    display: flex;
-    position: relative;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    padding-left: 4px;
-  }
-  .indicator {
-    display: flex;
-    justify-content: center;
-    text-align: center;
-    position: absolute;
-    margin: 0 auto;
-    user-select: none;
-    font-family: ${styles.MAIN_FONT};
-    font-weight: 600;
-    font-size: 15px;
-    line-height: 24px;
-    color: ${styles.MAIN_FONT_COLOR};
-  }
-  .red-indicator {
-    color: #ba4f4f;
-  }
-`;
+export { ChartWrapper } from "./ChartWrapper";
+export type { ChartWrapperProps } from "./ChartWrapper";
```

Comment:
why not just `export * from "./ChartWrapper"` ?

Reply:
I had originally been explicit with all exports, but missed this file when updating!